src/persist-client/README.md

```diff
@@ -85,3 +85,34 @@ cargo bench -p mz-persist-client --bench=benches
 Larger writes are expected to take the above latency floor plus however long it
 takes to write the extra data to e.g s3. TODO: Get real numbers for larger
 batches.
+
+## Tracing
+
+Persist offers introspection into performance and behavior through integration
+with the [tracing] crate.
+
+[tracing]: https://docs.rs/tracing
+
+There are many policies we could adopt for what is instrumented and at what
+level. At the same time, it's early enough that there are lots of unknowns
+around what will be most useful for debugging real problems and what knobs are
+wanted to opt into additional detail. As a result, we adopt a common idiom of
+tracing at the persist API boundary with the rest of mz (as well as persist's
+API boundary with external systems such as s3 and aurora, which we wouldn't do
+if they themselves offered tracing integration). Concretely:
+
+- All persist spans have a `target` of `persist`, regardless of which crate or
+  module they are in. In the future, we'll use `persist::foo` to allow fine
+  grained opt-in of supplementary info for targeted problems (e.g. unexpected
+  retries, slow throughput).
+- All persist spans have a `shard` field.
+- Every method in the persist public API is traced at info level. However, this
+  excludes "sugar" methods (e.g. WriteHandle::append and
+  WriteHandle::append_batch) that are implemented entirely in terms of other
+  public persist API methods.
+- All writes to external systems are traced at info level. Reads are traced at
```

Comment:
are you talking about the `event`'s or the spans? the level doesnt matter for opentel, as long as its above `debug`, but it does matter for stderr logging

Reply:
The reason it is `debug` is to avoid printing opentel span information in events in the common-case stderr logs, there is no real reason past that!

Reply:
okay adjusted this a bit. upgraded s3 reads to opentel-worthy